app/next-env.d.ts

```diff
@@ -0,0 +1,2 @@
+/// <reference types="next" />
```

Comment:
Are these supposed to be commented out?

Reply:
i tried deleting this and nextjs auto generates it on build if you don't already have one defined